src/lib/discord.ts

```diff
@@ -0,0 +1,15 @@
+import { httpClient } from '@/lib/httpClient'
+
+export async function discordNotification(message: string, link?: string) {
```

Comment:
**nitpick:** Let's parametrize this function a bit, and make these arguments more descriptive. We also should build message format before we pass it to not hardcode message format. I would also parametrize wehbookURL so we can use this function to send notifications to multiple channels. I would go with something like:

```ts
export async function sendDiscordNotificationToWebhook(params: {
message: string;
webhookUrl: string;
}) {
 await httpClient.post(webhookUrl, {
      content: message,
})
}
```

and if you don't want to copy-paste webhook url each time, then you can do:

```ts
export async function sendDiscordNotificationToModeratorChannel(message: string) {
  sendDiscordNotificationToWebhook({
    webhookUrl: process.env.TEST_WEBHOOK,
    message,
  })
}
```